sdk/include/opentelemetry/sdk/common/global_log_handler.h

```diff
@@ -29,16 +30,19 @@ namespace internal_log
 
 enum class LogLevel
 {
-  Error = 0,
-  Warning,
-  Info,
-  Debug
+  None    = OTEL_INTERNAL_LOG_LEVEL_NONE,
```

Comment:
Can we map this internal log level to the `Severity` in our public header?

Reply:
To the contrary, internal logs should be clearly separated from the log signal, to avoid confusion.

What this patch does is make the previous enum numbering more explicit, because the code already assumes that the value for each enum corresponds to the defines used.